rails/app/controllers/external_activities_controller.rb

```diff
@@ -170,71 +145,6 @@ def create
     end
   end
 
-  # PUT /external_activities/1
-  # PUT /external_activities/1.xml
-  def update
```

Comment:
I think this will be needed so the portal-settings form still works, or perhaps not?

Reply:
I tested this locally and when I try to save from the portal settings edit page, it does PUT to:
http://app.rigse.docker/eresources/:id
and gives a route not found.
I guess we don't really have any cucumber tests which exercise this form since all of the tests are passing.

Reply:
Ah, OK. The portal settings form was working for me locally after these changes, but then I restarted the Docker instance and it stopped working. I put that part back.